src/frameworks/drupal7/faq.md

```diff
@@ -19,22 +19,9 @@ deployment hooks to automatically run the updates.
 
 ## How can I override the default robots.txt?
 
-If your project is using a make file, you will end up with the default
-`robots.txt` provided by Drupal.
+Platform.sh allows the use of a custom `robots.txt` file on production. For pre-production environments we provide a [default that blocks all search engines](/administration/web/configure-environment.html#robotstxt) (to keep pre-production sites out of search indexes).
```

Comment:
Not 100% true about "production" vs "pre-production" (the truth being: the custom `robots.txt` would work any environment where `restrict_robots` is false, which is the default for just `master`)